HackIllinois/UI/TableView/Cells/HIBubbleCell/HIEventCell.swift

```diff
@@ -76,73 +76,64 @@ extension HIEventCell {
 extension HIEventCell {
     static func heightForCell(with event: Event, width: CGFloat) -> CGFloat {
         let heightFromEventName = HILabel.heightForView(text: event.name, font: HIAppearance.Font.eventTitle, width: width - 98)
-        let heightFromEventInfo = HILabel.heightForView(text: trimText(text: event.info, length: getMaxDescriptionTextLength()), font: HIAppearance.Font.eventDetails, width: width - 100)
+        let heightFromEventInfo = HILabel.heightForView(text: trimText(text: event.info, length: HIConstants.MAX_EVENT_DESCRIPTION_LENGTH), font: HIAppearance.Font.eventDetails, width: width - 100)
         let height = heightFromEventName + heightFromEventInfo + 90 + 15
-        if !event.sponsor.isEmpty {
-            return height + 21
-        }
         return height
     }
 
     static func <- (lhs: HIEventCell, rhs: Event) {
         lhs.favoritedButton.isActive = rhs.favorite
         var contentStackViewHeight: CGFloat = 0
-
+        // Add event title
         let titleLabel = HILabel(style: .event)
         titleLabel.numberOfLines = 0
         titleLabel.text = rhs.name
         lhs.contentStackView.addArrangedSubview(titleLabel)
-
+        let titleHeight = HILabel.heightForView(text: rhs.name, font: HIAppearance.Font.eventTitle, width: lhs.contentView.frame.width - 98)
+        contentStackViewHeight += titleHeight
+        // Add event type
+        let eventTypeLabel = HILabel(style: .eventType)
+        eventTypeLabel.text = rhs.eventType.lowercased().capitalized
+        contentStackViewHeight += eventTypeLabel.intrinsicContentSize.height
+        lhs.contentStackView.addArrangedSubview(eventTypeLabel)
+        // Create the container for showing time and points
+        let timePointsContainer = HIView()
+        timePointsContainer.constrain(height: 30)
+        let timeImageView = UIImageView(image: #imageLiteral(resourceName: "Clock"))
         let timeLabel = HILabel(style: .eventTime)
         timeLabel.text = Formatter.simpleTime.string(from: rhs.startTime) + " - " + Formatter.simpleTime.string(from: rhs.endTime)
-        lhs.contentStackView.addArrangedSubview(timeLabel)
-        lhs.contentStackView.setCustomSpacing(10, after: timeLabel)
-
-        if !rhs.sponsor.isEmpty {
-            let sponsorLabel = HILabel(style: .sponsor)
-            sponsorLabel.text = "Sponsored by \(rhs.sponsor)"
-            contentStackViewHeight += sponsorLabel.intrinsicContentSize.height
-            lhs.contentStackView.addArrangedSubview(sponsorLabel)
-        }
```

Comment:
I think we should still keep the sponsor label in the event cell (between event type and event time). Be sure to update the stack view height and cell height to account for this additional label.